### DISCUSSION TITLE
Jump to next / previous change (vcs)

### DISCUSSION THREAD

**Discussion by raphael-gruber-form3** [zed-industries/zed#6638] [Feature Requests]
Hey, it's super handy that you can see diffs highlighted in the scrollbar. Is there a way / is it planned to jump to the next/previous VCS change in the current buffer via a command? I would use that a lot!

**Comment by JosephTLyons** [comment]
Hey @raphael-gruber-form3, in the command palette, we have:

- `editor: go to hunk`
- `editor: go to prev hunk`

These should do what you need 👍. Let me know if not!

**Reply by raphael-gruber-form3** [reply]
exactly what i wanted, thanks!
